src/Symfony/Component/Notifier/Bridge/Twitter/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/twitter-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Twitter Notifier Bridge",
+    "keywords": ["twitter", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
+        "symfony/http-client": "^5.4|^6.0",
+        "symfony/notifier": "^6.2"
```

Comment:
shouldn't this require the mime component as well, as it uses it to represent attachments ?

Reply:
Mime is only needed when attaching a media, so it's optional.

Other comments addressed, thanks for the quick review!

Reply:
shouldn't it at least be a dev requirement ? Or is this untested ?

Reply:
:see_no_evil:

Reply:
tests added for uploading image+caption and video+subtitles